lib/device_api/android/device.rb

```diff
@@ -35,7 +35,8 @@ def status
         {
             'device' => :ok,
             'no device' => :dead,
-            'offline' => :offline
+            'offline' => :offline,
+            'unauthorized' => :unauthorized
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.